src/StackAdmin/Azs.Subscriptions/Tests/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.AzureStack.Management.Subscription" version="0.1.0-preview" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.6.0" targetFramework="net45" />
- 
-</packages>
\ No newline at end of file
+  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.7.2" targetFramework="net452" />
```

Comment:
Do we need these  refrences here? since we should refer to common runtime dependencies

Reply:
Probably not.